modules/nw-sriov-hwol-adding-network-attachment-definitions-to-pods.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="adding-network-attachment-definition-to-pods_{context}"]
+= Adding the network attachment definition to your pods
+
+After you create the machine config pool, the `SriovNetworkPoolConfig` and `SriovNetworkNodePolicy` custom resources, and the network attachment definition, you can apply these configurations to your pods by adding the network attachment definition to your pod specifications.
+
+.Procedure
+
+. In the pod specification, add the `.metadata.annotations.k8s.v1.cni.cncf.io/networks` field and specify the network attachment definition you created for hardware offloading:
```

Comment:
```suggestion
* In the pod specification, add the `.metadata.annotations.k8s.v1.cni.cncf.io/networks` field and specify the network attachment definition you created for hardware offloading:
```
Single-step procedures should use an unordered list.

Reply:
Thanks for catching that! Fixed!